build.gradle

```diff
@@ -422,7 +422,7 @@ project (':quasar-core') {
         jdk8test {
             java {
                 srcDir 'src/test/java'
-                srcDir 'src/jdk8Test/java'
+                srcDir 'src/jdk8test/java'
```

Comment:
Well spotted. This must mean that we haven't been running the JDK8 tests recently...

Reply:
Thanks! On an OSX platform would not build correctly, took me a while to figure out what was wrong.